pkg/features/feature.go

```diff
@@ -26,7 +25,7 @@ func (f *feature) Default() bool {
 }
 
 func (f *feature) Enabled() bool {
-	if buildinfo.ReleaseBuild {
+	if f.unchangeable {
```

Comment:
```suggestion
	if buildinfo.ReleaseBuild && f.unchangeable {
```
Without this change, I don't think `unchangeable` flags could be changed on dev builds, either.

Reply:
Wow I added the test case to test this (manually) and still I didn't check the unchangeable type 🤦 . Fixed now, thanks!